tests_e2e/test_selenium_trello.py

```diff
@@ -0,0 +1,92 @@
+import os
+import time
+from threading import Thread
+from flask import Flask, render_template, request, redirect, url_for, json
+import requests
+import unittest
+from unittest.mock import patch
+import pytest
+import pytest_html
```

Comment:
This import is unused, but also breaks the tests because `pytest-html` isn't listed in your project dependencies (pyproject.toml)

Similarly for the `requests_mock` and `requests_mock_flask` imports in the integration test.